cmd/minikube/cmd/mount.go

```diff
@@ -225,3 +251,45 @@ func getPort() (int, error) {
 	defer l.Close()
 	return l.Addr().(*net.TCPAddr).Port, nil
 }
+
+func enableCifsShare(hostShareName string, vmDestinationPath string, runner command.Runner) (error) {
+	// Ensure that the current user is administrator because creating a SMB Share requires Administrator privileges.
+	_ , err := pkghyperv.IsWindowsAdministrator()
+	if err != nil {
+		return err
+	}
+
+	hostname, _ := os.Hostname()
+	user, err := pkghyperv.GetCurrentWindowsUser()
+	if err != nil {
+		return err
+	}
+	user = strings.Replace(user,(hostname + "\\"), "",1)
+
+	// Check if the Credential exists in the credential store.
+	var credentialName = "minikube"
+	var password = ""
+	cred, err := wincred.GetGenericCredential(credentialName)
+	if err == nil {
+		out.T(out.Notice,"Credential {{.credential}} was found in the Windows Credential Store. Using that...",out.V{"credential":credentialName})
+		password = string(cred.CredentialBlob)
+	} else {
+		out.T(out.Enabling,"Please Type in the password for the user - [{{.username}}]",out.V{"username":user})
+		inputPassword, _ := terminal.ReadPassword(int(os.Stdin.Fd()))
+
+		cred := wincred.NewGenericCredential(credentialName)
+		cred.CredentialBlob = []byte(inputPassword)
+		wincrederr := cred.Write()
+		if wincrederr != nil {
+			return wincrederr
+		}
+		password = string(inputPassword)
+	}
+	mountCmd := fmt.Sprintf("sudo mkdir -p %s && sudo mount.cifs //%s/%s %s -o username=%s,password=%s,domain=%s",vmDestinationPath, hostname, shareName, vmDestinationPath, user, password, hostname)
+	err = cluster.MountCifs(runner, vmDestinationPath, mountCmd)
+	if err != nil {
+		out.ErrT(out.FailureType, "Failed mounting: {{.error}}", out.V{"error": err})
```

Comment:
Either display the error, or return it, but not both. Doing so results in duplicated error messages being shown.